samples/KafkaFlow.Sample.SchemaRegistry/Handlers/AvroMessageHandler.cs

```diff
@@ -1,9 +1,10 @@
-﻿namespace KafkaFlow.Sample.SchemaRegistry.Handlers;
+﻿using KafkaFlow.Middlewares.TypedHandler;
```

Comment:
Reorganize the usings please to include them inside the namespace

Reply:
Done